src/html5/parser.rs

```diff
@@ -3721,13 +3752,27 @@ impl<'chars> Html5Parser<'chars> {
         }
     }
 
+    fn parser_data(&self) -> ParserData {
+        let namespace = self
+            .get_adjusted_current_node()
```

Comment:
this implemented is not properly, the test case html5test-com.dat case 14 failed is because this.

Reply:
👍